packages/swirl-components/src/components/swirl-card/swirl-card.css

```diff
@@ -25,6 +25,32 @@
     0 0.25rem 1rem rgba(25, 26, 28, 0.04);
 }
 
+.content-section {
+  padding: 0 var(--s-space-16);
+}
+
+.image-section {
+  & ::slotted(img) {
+    width: 100%;
```

Comment:
**suggestion:** We could add the following here:

```css
max-height: 100%;
object-fit: cover;
```

And add an optional `imageAspectRatio` prop to the component:

```jsx
<div
  class="image-section"
  style={{ aspectRatio: this.imageAspectRatio }}
>
```

This way we would be able to control the aspect ratio of images that have different sizes.